src/main/java/com/amazon/opendistroforelasticsearch/sql/rewriter/nestedfield/Identifier.java

```diff
@@ -51,8 +51,13 @@ void rewrite(Scope scope) {
         }
     }
 
+    /**
+     * return the path of the expr name. e.g.
+     * if (expr.name = p.name), return p
```

Comment:
maybe better if we clarify the sample SQL, ex. "expecting `p` returned as path in both `WHERE p.name = 'A'` and `WHERE p IS NULL` cases, in which `expr.name` = `p.name` and `p` separately".

Reply:
Done.